DEVELOPMENT.md

```diff
@@ -171,6 +171,9 @@ A test-run with `kind` could look like this:
 # create kind cluster
 kind create cluster --config .github/kind-cluster-<k8s-version>.yaml
 
+# load the image to the kind cluster
```

Comment:
I added this because I did this when I was testing. But I see your point, lets keep it clean and let the docs focus on a global procedure. I will remove this comment from the docs.